src/components/logpile.vue

```diff
@@ -139,16 +146,46 @@ export default {
           svg_add
             .selectAll(".bar")
             .data(data, function(d) { return d.Type })
-            .enter()
+          .enter()
             .append("rect")
             .classed("bar", true)
             .attr("class", d => { return "bar " + d.Type }) // to grab in interaction
             .attr("x", d => xScale(x(d)))
             .attr("y", y_height)
-            .attr("width", 2)
+            .attr("width", 10)
             .attr("height", y_height)
-            .attr("opacity", 0.85)
-            .attr("fill", "pink")    // TODO: color based on type
+            .attr("fill", "royalblue")
+
+            var file = 'salineLake.jpg';
+
+          svg_add
+            .selectAll(".bar")
+            .on("mouseover", function(d) {		
```

Comment:
And eventually I think it would be good to pull this interaction code out into a function, to keep the chart creation separate from the interaction code (particularly if the interaction code gets more complex), but since this is just a quick exploration I don't think the change needs to be made now.

Reply:
added